src/TicTacToe/Board.java

```diff
@@ -15,16 +15,18 @@ public void displayBoard() {
         System.out.println();
     }
 
-    public void putToBoard(char position, char sign) {
+    public boolean putToBoard(int position, char sign) {
 
         for (int i = 0; i < board.length; i++) {
             for (int j = 0; j < board[i].length; j++) {
-                if (board[i][j] == position) {
+                if (board[i][j] == (char) (48 + position)) {
```

Comment:
Please put '(char) (48 + position)' to static final field, name it properly and use it here. If somebody look at this code and doesn't know ASCII he will be confused.